frontend/src/common/components/footer/Footer.tsx

```diff
@@ -54,6 +56,9 @@ export const Footer = () => {
               Contact Us
             </a>
           </li>
+          <li>
```

Comment:
The styling looks weird. I think this should be ```<li><a>Build: {prNumber}</a></li>``` instead, since the styling current only applies for ```<a>``` elements